firmware/application/apps/ui_looking_glass_app.cpp

```diff
@@ -153,13 +153,13 @@ void GlassView::on_range_changed()
     max_power = 0;
     bins_Hz_size = 0;                               //reset amount of Hz filled up by pixels
     
-    baseband::set_spectrum(SEARCH_SLICE_WIDTH, field_trigger.value());   
+    baseband::set_spectrum(LOOKING_GLASS_SLICE_WIDTH, field_trigger.value());   
     receiver_model.set_tuning_frequency(f_center_ini); //tune rx for this slice
 }
 
-void GlassView::PlotMarker(rf::Frequency pos)
+void GlassView::PlotMarker(rf::Frequency fpos)
 {
-    pos = pos * MHZ_DIV;
+    int64_t pos = fpos * MHZ_DIV;
```

Comment:
is this the only line useful in the whole PR?

Reply:
Yes, that and the custom define for looking glass which cause an unnecessary warning and can lead to future errors.

Reply:
Talk on discord on the subject led to the following changes instead:
portapack::display.fill_rectangle({(int)pos - 2, 100, 5, 3}, Color::red()); //Red marker middle
portapack::display.fill_rectangle({(int)pos - 1, 103, 3, 3}, Color::red()); //Red marker middle
portapack::display.fill_rectangle({(int)pos, 106, 1, 2}, Color::red()); //Red marker middle
Who is doing the PR update ?

Reply:
Done

Reply:
OK cool. I do not think refactoring code is a good thing if there is no reason by the way. I will clean the PR later, since is simple enough 👍